packages/compass-e2e-tests/helpers/commands/expand-fle-local-kms.ts

```diff
@@ -0,0 +1,21 @@
+import type { CompassBrowser } from '../compass-browser';
+import * as Selectors from '../selectors';
+
+export async function expandFLELocalKMS(
```

Comment:
Nice! this looks super handy .. does it makes sense to turn this into `expandAccordion(selector)` and make it usable for every other section too?